cumulusci/robotframework/tests/cumulusci/base.robot

```diff
@@ -64,6 +64,51 @@ Test Perf - Parser does not choke with no keywords - Should Fail
 Test Perf Measure Other Metric
     Set Test Metric    Max_CPU_Percent    30
 
-Test Should Generate An Error - Mismatched End Perf Timer - Should Fail
-    [Tags]      noncritical
-    End Perf Timer
+Mismatched End Perf Timer - Should Fail
+    Run Keyword and Expect Error
+    ...         *Elapsed time clock was not*
+    ...         End Perf Timer
+
+Test Elapsed Time For Last Record
+    # This test uses contacts as if they were "jobs" becaues they are
+    # easy to insert. I don't currently have a better alternative 
+    # for a job-like objects which is easy to create in a vanilla
+    # SF org
+    ${contact_id} =  Salesforce Insert  Contact  FirstName=Dummy1  LastName=Dummy2
+    sleep   1
```

Comment:
Is this sleep necessary? Is it because it takes the Salesforce API a second or two to register the change? Is there any way to be smarter about this than a hard-coded sleep?

Reply:
Removed.